pallets/gear/rpc/src/lib.rs

```diff
@@ -336,6 +353,22 @@ where
             .map(Bytes)
     }
 
+    fn read_state_batch(
+        &self,
+        batch_id_payload: Vec<(H256, Bytes)>,
```

Comment:
Opened an issue to max this usage safe as well #3210.
At this moment I'd hardcode some const